WorkflowSwiftUI/Tests/ObservableScreenTests.swift

```diff
@@ -0,0 +1,80 @@
+#if canImport(UIKit)
+
+import SwiftUI
+import ViewEnvironment
+@_spi(ViewEnvironmentWiring) import ViewEnvironmentUI
+import XCTest
+@testable import WorkflowSwiftUI
```

Comment:
@watt The `@testable` was required here because the `StateAccessor` init is not public. Not sure if that was intentional or not. I can open it up as part of this if you'd like!

Reply:
hmm, probably OK if it's public.

Reply:
👍🏽 https://github.com/square/workflow-swift/pull/297/commits/46961b0924ae027cd40aa1a8197fc86abab9fe56